comit/src/lib.rs

```diff
@@ -15,6 +15,8 @@
 #![type_length_limit = "1049479"] // Regressed with Rust 1.46.0 :(
 
 pub mod actions;
+#[cfg(feature = "quickcheck")]
```

Comment:
As this is test code I would expect it to be behind `cfg(test)`

Reply:
If it is behind `cfg(test)`, you cannot use it from a different crate. It is quite common for library crates to expose such implementations under the feature-flag of the library that enables them. It is similar to how implementations of `serde::Serialize` are exposed under a `serde` feature flag.

Reply:
> If it is behind cfg(test), you cannot use it from a different crate.

Can you clarify how it is behind `cfg(test)`?

Reply:
It isn't. If it were, then all consumers of the `comit` crate would not be able to use the functions in this module. This is the **comit** crate in case you didn't notice :)

Reply:
Are you saying that we don't need to expose this module publicly because neither `cnd` nor `nectar` are currently using the functions in this module?

Reply:
Ok, thinking about it. It's fine the way it is :)